packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Change Log - @typespec/http
 
+## 0.61.0
```

Comment:
I think there might have been an issue with running the release multiple times every package was increased by 3 version